### PR TITLE
Use `doc = include_str!` instead of `doc(include)`

### PR DESCRIPTION
**Pull request by taiki-e**

### CODE REVIEW

**Review by taiki-e — COMMENTED** [review]
`doc = include_str!` has a problem that displaying the wrong line number (https://github.com/rust-lang/rust/issues/81070), but it is stabilized anyway.

bors r+

File: tests/doc/lib.rs [hunk 1]
```diff
@@ -6,12 +6,11 @@
...
 
-// As `feature(external_doc)` and `doc-comment` do not work with `cfg(test)`,
+// As `doc = include_str!` and `doc-comment` do not work with `cfg(test)`,
 // and `cfg(doctest)` requires 1.40, these tests are split into this crate until
 // MSRV increases.
 
-#[cfg(doctest)]
-#[doc(include = "../../README.md")]
+// https://github.com/rust-lang/rust/issues/82768
+#[cfg_attr(doctest, cfg_attr(doctest, doc = include_str!("../../README.md")))]
```
Comment (Aaron1011) [inline tests/doc/lib.rs:15]: :laughing:

**Comment by taiki-e** [comment]
hmm gha incident... https://www.githubstatus.com/incidents/kjljm9gfpgxm

**Comment by taiki-e** [comment]
bors retry

**Pull request merged**